libfreerdp/crypto/test/TestKnownHosts.c

```diff
@@ -46,8 +46,9 @@ static int prepare(const char* currentFileV2, const char* legacyFileV2, const ch
 
 	for (i=0; i<sizeof(hosts)/sizeof(hosts[0]); i++)
 	{
-		fwrite(hosts[i], strlen(hosts[i]), sizeof(char), fl);
-		fwrite(hosts[i], strlen(hosts[i]), sizeof(char), fc);
+		if (fwrite(hosts[i], strlen(hosts[i]), sizeof(char), fl) != sizeof(char) ||
+			fwrite(hosts[i], strlen(hosts[i]), sizeof(char), fc) != sizeof(char))
+			goto finish;
```

Comment:
Not your code but the fwrite arguments (size and nmemb) seem to be mixed. Although it works like this the correct way would be:
```
size_t nmemb = strlen(hosts[i]);
size_t size = sizeof(char);
if (fwrite(hosts[i], size, nmemb, fl) != nmemb) ...
```
... because we want to write nmemb characters where each character has sizeof(char) bytes.